dnf-behave-tests/features/steps/cmd.py

```diff
@@ -59,6 +59,16 @@ def faketime_today(context, when):
     context.execute_steps('when I move the clock backward to "{}"'.format(when))
 
 
+@behave.step("I fake kernel release to {release}")
+def i_fake_kernel_release(context, release):
+    context.fake_kernel_release = "fakeuname {} ".format(release)
+
+
+@behave.step("I do not fake kernel release")
```

Comment:
How about calling the step _"I stop faking kernel release"_? Sounds a bit better to me.

Reply:
Ok, renamed.